maven-example-noncontainer/cloudbuild.yaml

```
@@ -0,0 +1,29 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+# [START cloudbuild_maven]
+steps:
+  - name: maven:3.3-jdk-8
+    entrypoint: mvn
+    args: ["test"]
+  - name: maven:3.3-jdk-8
+    entrypoint: mvn
+    args: ["package", "-Dmaven.test.skip=true"]
+  artifacts:
```

Marshalling error: `yaml: line 21: did not find expected '-' indicator`

```suggestion
artifacts:
```